configure.py

```diff
@@ -214,6 +214,33 @@
             Object(NonMatching, "quazal/unk_800A515C.cpp")
         ],
     },
+    {
+        "lib": "rb3",
+        "mw_version": "Wii/1.3",
+        "cflags": cflags_rb3,
+        "host": False,
+        "objects": [
+            Object(NonMatching, "rb3/unk_80357E10.cpp")
+        ],
+    },
+    {
+        "lib": "rb3",
+        "mw_version": "Wii/1.3",
+        "cflags": cflags_rb3,
+        "host": False,
+        "objects": [
+            Object(NonMatching, "rb3/unk_8037A49C.cpp")
+        ],
+    },
+    {
+        "lib": "rb3",
+        "mw_version": "Wii/1.3",
+        "cflags": cflags_rb3,
+        "host": False,
+        "objects": [
+            Object(NonMatching, "rb3/unk_8000D980.cpp")
+        ],
+    },
```

Comment:
move all of the object files to the first rb3 library and delete all of the extra ones
they should not be separate